echopype/calibrate/ek80_complex.py

```diff
@@ -12,22 +12,19 @@ def tapered_chirp(
     fs,
     transmit_duration_nominal,
     slope,
-    frequency_nominal=None,
-    frequency_start=None,
-    frequency_end=None,
+    transmit_type,
```

Comment:
seems like `transmit_type` is unused and can be removed

Reply:
Right, thanks! I'll include this in the next batch of commits.